WordPress/src/main/res/layout/post_list_author_selection.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+             xmlns:tools="http://schemas.android.com/tools"
+             android:layout_width="match_parent"
+             android:layout_height="match_parent">
+    <!-- FrameLayout required since we want to store a view holder it it's tag but Glide uses the tag on the ImageView -->
```

Comment:
It looks like there is a small spelling mistake in this comment. I'd suggest moving the comment above the `FrameLayout` to make it more contextual.

Reply:
Sure

Reply:
Fixed in cf7fef2fcf160c487c0a61cae955d65c5f83b569